src/styles/index.scss

```diff
@@ -2,2 +1,19 @@
+@import 'variables';
+@import 'header';
+@import 'nav';
+@import 'catalog';
+@import 'card';
+@import 'star';
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 400;
+  color: $main-color;
```

Comment:
Ensure that the font-family and color variables are defined in the 'variables.scss' file. If they are not, this could lead to unexpected results.